fca_import_scripts/README.md

```diff
@@ -0,0 +1,43 @@
+# FCA Import Scripts
+
+This is a temporary procedure for importing new FCA data into the lookup tables
+in the RAD database.
+
+## Steps
+
+1) Download the latest set of zip files from the FCA SFTP directory to `archives/`
+
+2) Run `./fca_data_to_sql.sh` and follow the prompts.
+
+---
+
+**`CSV::MalformedCSVError` errors** - You will might see the scripts fail out
+with these errors if the autorepair fails. Usually they are due to name fields
+that contain double quotes. To fix these wrap the entire field in quotes, then
```

Comment:
"To fix these warp the entire field in quotes ..." <-- this is no longer true now we're not using CSV as a parser.